src/components/icon-row/icon-row.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { PropTypes } from 'react';
+import { TextIcon } from 'nordnet-ui-kit';
+import RowInfo from './row-info';
+
+const IconRow = ({
+  tag,
+  iconComponent,
+  textIconText,
+  topLeftComponent,
+  bottomLeftComponent,
+  topRightComponent,
+  bottomRightComponent,
+  minHeight,
+  infoPaddingLeft,
```

Comment:
Can change these 4 lines to infoStyles (containing paddingLeft, paddingRight, marginRight, marginLeft)